eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import path from "node:path";
```

Comment:
Nit: we treat strings as single-quoted everywhere else, so should probably do that in here. Haha ironically I think `eslint` would have caught this except for the `.mjs` extension.